src/screens/ConfirmNotificationCodeScreen/ConfirmNotificationCodeScreen.tsx

```diff
@@ -121,13 +126,13 @@ class ConfirmNotificationCodeScreen extends PureComponent<Props, State> {
               onPress={this.onConfirm}
               disabled={userCode.length < CONST.codeLength}
             />
-            <FlatButton
+            {/* <FlatButton // uncomment when api for resend works
```

Comment:
TODO? When everyone follows one convention it's easier to find all the comments like this